testsuite/testsuite-microprofile-jwt/src/test/resources/project-default-groups.yml

```diff
@@ -0,0 +1,13 @@
+mp:
+  jwt:
+    verify:
+      publickey: MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAlivFI8qB4D0y2jy0CfEqFyy46R0o7S8TKpsx5xbHKoU1VWg6QkQm+ntyIv1p4kE1sPEQO73+HY8+Bzs75XwRTYL1BmR1w8J5hmjVWjc6R2BTBGAYRPFRhor3kpM6ni2SPmNNhurEAHw7TaqszP5eUF/F9+KEBWkwVta+PZ37bwqSE4sCb1soZFrVz/UT/LF4tYpuVYt3YbqToZ3pZOZ9AX2o1GCG3xwOjkc4x0W7ezbQZdC9iftPxVHR8irOijJRRjcPDtA6vPKpzLl6CyYnsIYPd99ltwxTHjr3npfv/3Lw50bAkbT4HeLFxTx4flEoZLKO/g0bAoV2uqBhkA9xnQIDAQAB
+      issuer: "http://testsuite-jwt-issuer.io"
+smallrye:
+  jwt:
+    claims:
+      groups: User
```

Comment:
What's the purpose of the `mp.*` and `smallrye.*` settings and can't they not be set under `thorntail.*`?

Reply:
I wanted to use and also check that the standard MP JWT properties are effective, they are in smallrye-jwt code so I thought it would be to check them out. The new 'smallrye-jwt' properties have no yet MP JWT analog but we have discussed with @starksm64 that the equivalent MP JWT props should also be available, so before starting adding even more properties in Thorntail it is worth waiting a bit

Reply:
Re the new `smallrye` properties, we can have thorntail equivalents but smallrye-jwt 1.1.4 is out now and 1.1.5-SNAPSHOT will need to be tweaked a bit to support the case where a (Thorntail) extension submits its own properties into the smallrye-jwt code setting up the JWAuthContextInfo. But again, may be it is worth waiting a bit